### PR TITLE
add support for snake_case for contentAssetId in Adobe Heartbeat

### PR DESCRIPTION
**Pull request by rhall-twilio**
**What does this PR do?**
* Drop support for `assetId` for `StandardVideoMetadata` in Adobe Heartbeat
* Add support for `content_asset_id` for `StandardVideoMetadata` in Adobe Heartbeat

**Are there breaking changes in this PR?**

The removal of `assetId` from `StandardVideoMetadata` cannot be considered a breaking change given that the value was always overriden by `content_asset_id`, even if undefined. See for more details:  https://segment.atlassian.net/browse/STRATCONN-1491

**Testing**

Testing completed successfully using the `tester` page by stepping through debugger on the `createStandardVideoMetadata` function and making sure that

 ```analytics.track("video playback started", {assetId: "content-asset-id"})```

did not populate the media asset id while

 ```analytics.track("video playback started", {content-asset-id: "content-asset-id"})```

did successfully so

Testing completed successfully using unit tests.

**Any background context you want to provide?**
https://segment.atlassian.net/browse/STRATCONN-1491

**Is there parity with the server-side/android/iOS integration components (if applicable)?**
N/A

**Does this require a new integration setting? If so, please explain how the new setting works**
N/A

**Links to helpful docs and other external resources**
https://segment.com/docs/connections/destinations/catalog/adobe-analytics/heartbeat/#video-playback-started

### CODE REVIEW

File: integrations/adobe-analytics/lib/index.js
Comment (kdaswani) [inline integrations/adobe-analytics/lib/index.js:1504]: Do this logic prioritize `content_asset_id` or `contentAssetId` first? I think we want to always prioritize `contentAssetId` as to not break any existing implementation (i.e. if the customer happens to have both snake and camel case in their event - unlikely but good to be safe). Fallback on `content_asset_id`.

Comment (rhall-twilio) [reply integrations/adobe-analytics/lib/index.js:1504]: It is currently prioritizing `content_asset_id`. I will make the correction.

**Review by brennan — APPROVED** [review]

File: integrations/adobe-analytics/lib/index.js
Comment (kdaswani) [reply integrations/adobe-analytics/lib/index.js:1504]: The test below makes me think that maybe these got swapped. Can we swap them back @rhall-twilio? Otherwise all looks good to me! Will wait for local testing to be complete and review from another eng before approving. Thanks!

**Review by kdaswani — CHANGES_REQUESTED** [review]

File: integrations/adobe-analytics/lib/index.js
Comment (kdaswani) [reply integrations/adobe-analytics/lib/index.js:1504]: So fast at responding @rhall-twilio 😂

Comment (rhall-twilio) [reply integrations/adobe-analytics/lib/index.js]: This is now resolved.

**Review by kdaswani — APPROVED** [review]
ty robert! looks great

**Pull request merged**
